Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -258,6 +258,14 @@ extension BrowserViewController: WKNavigationDelegate {
                 self.tabManager.selectedTab?.alertShownCount = 0
                 self.tabManager.selectedTab?.blockAllAlerts = false
             }
+            
+            // Prevents synthetically activated links such as: CVE-2017-7089
+            //Follow desktop
+            if url.scheme == "data" {
```

Comment:
will this also block `window.open('data:...')`?

Reply:
@Brandon-T ^

Reply:
@diracdeltas I have implemented: https://blog.mozilla.org/security/2017/11/27/blocking-top-level-navigations-data-urls-firefox-59/

It does indeed block `window.open`. I tested against the latest Firefox-Quantum to make sure our behaviour is the same.